modules/cli/cli.go

```diff
@@ -167,6 +171,9 @@ func (c *CLI) Parse(arguments []string) manager.Config {
 		anonymousReportsEnabled = false
 	}
 
+	webhookCertificateConfigBaseImage := c.deferFlagValues.WebhookCertificateConfigBaseImage
+	webhookCertificateConfigShellImage := c.deferFlagValues.WebhookCertificateConfigShellImage
```

Comment:
You shouldn't really need this here. Those 2 flags can simply be used as is.

This func is for flags that require more logic than just binding to a variable.